services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js

```diff
@@ -113,6 +113,10 @@ qx.Class.define("osparc.desktop.StudyEditor", {
     }
   },
 
+  statics: {
+    READ_ONLY_TEXT: qx.locale.Manager.tr("You do not have writing permissions.<br>Your changes will not be saved.")
```

Comment:
Instead of `Your changes will not be saved.` I would write something like `No changes can be made and no changes will be saved`.

The message will make it clear why I can't add or change anything.

Reply:
This is coming from the App Team.